infra/modules/network/flow-logs.tf

```diff
@@ -0,0 +1,69 @@
+# File contents are based on the following example:
+# https://github.com/terraform-aws-modules/terraform-aws-vpc/blob/master/examples/vpc-flow-logs/main.tf
+
+# https://registry.terraform.io/providers/hashicorp/random/latest/docs/resources/pet
+resource "random_pet" "this" {
+  length = 3
+}
+
+# docs: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/cloudwatch_log_group
+resource "aws_cloudwatch_log_group" "flow_log" {
+  name = "vpc-flow-logs-to-cloudwatch-${random_pet.this.id}"
```

Comment:
Why do we need the name randomized? Would using `name_prefix` instead of `name` be another way?

Reply:
This was a copy paste - you're right that we don't need this randomized per-se. I'll check if there's a name prefix...

Reply:
Updated! https://github.com/HHS/simpler-grants-gov/pull/1089/commits/3a090cce230ad7701a5e604c6b02fa90d1e56c83